cuda/common/include/pcl/cuda/cutil_inline_drvapi.h

```diff
@@ -115,9 +120,9 @@ inline int cutilDrvGetMaxGflopsDeviceId()
         cutilDrvSafeCallNoSync (cuDeviceGetAttribute (&major, CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR, current_device));
         cutilDrvSafeCallNoSync (cuDeviceGetAttribute (&minor, CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MINOR, current_device));
 
-		if (major == 9999 && minor == 9999) {
-		    sm_per_multiproc = 1;
-		} else {
+        int sm_per_multiproc = 1;
+
+		if (major != 9999 || minor != 9999) {
```

Comment:
This block should have the same indentation as the `int sm_per_multiproc = 1;`

Same below

Reply:
Are these files following the PCL style guide? I did "no tabs, 2 spaces" and results vary. Might be due to editor issues, I tried to use different ones.

Can I run our clang-format on these files, for uniformity as a separate commit? 
Trying different editors just to solve this problem is driving me crazy :laughing: 

Also, does local diff and github diff differ? [link](https://github.com/isaacs/github/issues/717).

Reply:
> for uniformity as a separate commit?

Try a different PR. Add to the whitelist in `.dev/format.sh`

Reply:
> This block should have the same indentation as the `int sm_per_multiproc = 1;`
> 
> Same below

Done